js/ppom-simple-popup.js

```diff
@@ -64,7 +64,7 @@
 	var prefix = 'ppom-popup';
 
 	function pluploadRefresh() {
-		if( ! uploaderInstances ) {
+		if( ! 'object' !== typeof uploaderInstances ) {
```

Comment:
I'm not sure about the double usage of `!`. Should it not be like this if we want to filter out non-object values?

```javascript
 if ( typeof uploaderInstances !== 'object'   ) {
        return;
 }
```

In your case, the function returns early if the `uploaderInstances` is an object.

Reply:
@Soare-Robert-Daniel I missed to remove it